data/presets/amenity/public_bookcase.json

```diff
@@ -3,23 +3,28 @@
     "fields": [
         "name",
         "public_bookcase/type",
+        "books",
```

Comment:
I would have said that for public bookcases, it is rather the exception to find only specific types of books.  For this reason, I would have put this field to the `moreFields` section.

Reply:
It is already exception they are mapped, let alone such tags being discovered by users, but it is fine to change it.